pocketimdb/movies/serializers.py

```diff
@@ -1,8 +1,17 @@
 from rest_framework import serializers
-from .models import Movie
+from .models import Movie, MovieLike
 
 
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ['id', 'title', 'description', 'cover_image_url', 'genre']
\ No newline at end of file
+        fields = ['id', 'title', 'description', 'cover_image_url', 'genre', 'likes', 'dislikes', 'user_liked_or_disliked']
```

Comment:
ovaj meta uvijek ide na kraj, a ova polja na pocetku, to je jednostavno nacin na koji pisemo, ne posotoji bolje rjesenje, ovo je cisto style guide, pa hajde da ga postujemo